conf/app.routes

```diff
@@ -15,7 +15,7 @@ GET        /timeout
 #################       Summary pages      #########################################
 
 GET        /:taxYear/employment-summary                                             controllers.employment.EmploymentSummaryController.show(taxYear: Int)
-POST       /:taxYear/employment-summary                                             controllers.employment.EmploymentSummaryController.submit(taxYear: Int)
+GET       /:taxYear/employment/add-new-employer                                         controllers.employment.EmploymentSummaryController.addNewEmployer(taxYear: Int)
```

Comment:
If we redirect from the view directly to the EmployerNameController we will not need that.

Reply:
As explained above, i think its still needed